templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -564,64 +530,29 @@
-                                                            pattern: '^[1-9][0-9]*$',
-                                                            value: proceduresettings.publicParticipationPhaseObject.iteration
-                                                        },
-                                                        id: 'r_public_participation_phase_iteration',
-                                                        type: 'text',
-                                                        required: true,
-                                                    }) }}
-                                                </div>
-                                            {% endif %}
-                                        </div>
-
-                                        {{ uiComponent('inline-notification', {
-                                            message: 'explanation.procedure.external.permissions'|trans ~ '<span data-procedure-permission-set></span>',
-                                            type: 'warning'
-                                        }) }}
-
-                                        {% if hasPermission('feature_auto_switch_to_procedure_end_phase') and not hasPermission('feature_auto_switch_procedure_phase') %}
-                                            <div
-                                                v-if="publicPhaseIsInParticipation"
-                                                class="lbl__hint u-mt-0_25 u-mb-0">
-                                                {{ 'period.autoswitch.hint'|trans({ phase: 'procedure.phases.external.evaluating'|trans }) }}
-                                            </div>
-                                        {% endif %}
-                                    </div>
+                                    {% set externalPhaseOptions = [] %}
+                                    {% for phase in templateVars.externalPhases %}
+                                        {% set externalPhaseOptions = externalPhaseOptions|merge([{
+                                            value: phase.key,
+                                            label: phase.name,
+                                            permissionset: phase.permissionset,
+                                        }]) %}
+                                    {% endfor %}
+                                    <participation-phases
+                                        autoswitch-hint="{{ 'period.autoswitch.hint'|trans({ phase: 'procedure.phases.external.evaluating'|trans }) }}"
+                                        field-name="r_publicParticipationPhase"
+                                        help-text="{{ 'text.procedure.edit.external.change_phase'|trans }}"
+                                        label-text="{{ "procedure.public.phase"|trans }}"
+                                        :participation-phases="['participation', 'earlyparticipation', 'anotherparticipation']"
```

Comment:
Is it correct to hard code the phases here?

Reply:
At least as correct as it was before.
See removed line templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig:112